Dockerfile

```diff
@@ -1,8 +1,16 @@
-FROM mas.ops.maap-project.org/root/jupyter-image/vanilla:develop
+FROM continuumio/miniconda3:4.7.12
 
-COPY requirements.txt requirements.txt
+RUN pip install --no-cache-dir --upgrade pip
+
+# install maap-py library
+ENV MAAP_CONF='/projects/maap-py/'
+RUN git clone --single-branch --branch master https://github.com/MAAP-Project/maap-py.git \
+    && cd maap-py \
+    && pip install --no-cache-dir -e .
 
-RUN pip install -r requirements.txt
+# install testing requirements
+COPY requirements.txt requirements.txt
```

Comment:
our requirements file installs the same version of ipykernel as the ADE/DPS image.